pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -518,6 +546,59 @@ func (a *cpuAccumulator) takeFullSockets() {
 		a.take(cpusInSocket)
 	}
 }
+func (a *cpuAccumulator) takeFullUncore() {
```

Comment:
please add a space to separate the functions.

Reply:
done